vividus-http-client/src/main/java/org/vividus/http/HttpRequestBuilder.java

```diff
@@ -86,7 +86,7 @@ public HttpRequestBuilder withHeaders(List<Header> headers)
         return this;
     }
 
-    public HttpRequestBase build()
```

Comment:
could you please clarify a reason behind the changes?

Reply:
https://github.com/vividus-framework/vividus/blob/master/vividus-plugin-rest-api/src/main/java/org/vividus/http/HttpRequestExecutor.java#L103 - the exception is caught here, but it's not thrown here, it"s thrown in some wrapped methods, it's really hard to trace unchecked exception with such approach